--- yiff-2.14.1/yiffconfig/yccb.c.orig	2003-06-04 09:04:10.000000000 +0200
+++ yiff-2.14.1/yiffconfig/yccb.c	2003-07-20 18:36:02.000000000 +0200
@@ -1304,7 +1304,7 @@
 "No Y Audio Mode selected!",
 "There is no Y Audio Mode currently selected to apply\n\
 values for.",
-"You must select a Y Audio Mode from the list and then
+"You must select a Y Audio Mode from the list and then\n\
 apply its values to the Y server.",
 		CDIALOG_ICON_ERROR,
 		CDIALOG_RESPONSE_OK | CDIALOG_RESPONSE_HELP,
--- yiff-2.14.1/yiffutils/ymixer.c.orig	2003-06-04 09:04:09.000000000 +0200
+++ yiff-2.14.1/yiffutils/ymixer.c	2003-07-20 18:38:41.000000000 +0200
@@ -27,7 +27,7 @@
 Usage: ymixer [mixer] [value1] [value2] [value3] [value4] [options]\n\
 \n\
     [mixer] specifies the mixer channel device name, running without\n\
-    any arguments prints a list of available mixer names.
+    any arguments prints a list of available mixer names.\n\
 \n\
     [value#] specify the percent values (from 0.0%%, 100.0%%) to be set.\n\
     If no [value#] is specified then the current value is printed.\n\
