--- yiff2.12.4/yiffconfig/Makefile.orig	Sat May  5 10:53:23 2001
+++ yiff2.12.4/yiffconfig/Makefile	Sun Feb  2 11:08:37 2003
@@ -117,7 +117,7 @@
 # Build Rules:
 #
 $(BIN): $(OBJ_C) $(OBJ_CPP)
-	$(CC) $(OBJ_C) $(OBJ_CPP) -o $(BIN) $(LIB) $(LIB_DIR)
+	$(CPP) $(OBJ_C) $(OBJ_CPP) -o $(BIN) $(LIB) $(LIB_DIR)
 
 all: $(BIN)
 
--- yiff2.12.4/yiffutils/Makefile.orig	Sat May  5 10:53:22 2001
+++ yiff2.12.4/yiffutils/Makefile	Sun Feb  2 11:12:31 2003
@@ -110,42 +110,42 @@
 	helloworld.c -o helloworld
 
 yaudiocd:
-	$(CC) $(CFLAGS) $(INC) $(LIB_DIR) $(LIB) \
+	$(CPP) $(CFLAGS) $(INC) $(LIB_DIR) $(LIB) \
 	string.cpp \
 	yaudiocd.c -o yaudiocd
 
 yclientmessage:
-	$(CC) $(CFLAGS) $(INC) $(LIB_DIR) $(LIB) \
+	$(CPP) $(CFLAGS) $(INC) $(LIB_DIR) $(LIB) \
 	string.cpp \
         yclientmessage.c -o yclientmessage
 
 yhost:
-	$(CC) $(CFLAGS) $(INC) $(LIB_DIR) $(LIB) \
+	$(CPP) $(CFLAGS) $(INC) $(LIB_DIR) $(LIB) \
 	string.cpp \
 	yhost.c -o yhost
 
 ymixer:
-	$(CC) $(CFLAGS) $(INC) $(LIB_DIR) $(LIB) \
+	$(CPP) $(CFLAGS) $(INC) $(LIB_DIR) $(LIB) \
 	string.cpp \
 	ymixer.c -o ymixer
 
 yplay:
-	$(CC) $(CFLAGS) $(INC) $(LIB_DIR) $(LIB) \
+	$(CPP) $(CFLAGS) $(INC) $(LIB_DIR) $(LIB) \
 	disk.cpp string.cpp \
 	yplay.c -o yplay
 
 yrecinfo:
-	$(CC) $(CFLAGS) $(INC) $(LIB_DIR) $(LIB) $(STDSRC) \
+	$(CPP) $(CFLAGS) $(INC) $(LIB_DIR) $(LIB) $(STDSRC) \
 	string.cpp \
 	yrecinfo.c -o yrecinfo
 
 yset:
-	$(CC) $(CFLAGS) $(INC) $(LIB_DIR) $(LIB) $(STDSRC) \
+	$(CPP) $(CFLAGS) $(INC) $(LIB_DIR) $(LIB) $(STDSRC) \
 	string.cpp \
 	yset.c -o yset
 
 yshutdown:
-	$(CC) $(CFLAGS) $(INC) $(LIB_DIR) $(LIB) $(STDSRC) \
+	$(CPP) $(CFLAGS) $(INC) $(LIB_DIR) $(LIB) $(STDSRC) \
 	string.cpp \
 	yshutdown.c -o yshutdown
 
--- yiff-2.14.5/libY2/Makefile.orig	2005-09-26 10:41:57.000000000 +0200
+++ yiff-2.14.5/libY2/Makefile	2006-04-23 19:32:25.030530000 +0200
@@ -139,7 +139,7 @@
 	+$(CPP) -c $*.cpp $(INC_DIRS) $(CFLAGS) $(CPPFLAGS)
 
 $(LIB): $(OBJ_C) $(OBJ_CPP)
-	$(CC) $(OBJ_C) $(OBJ_CPP) -Wl,-soname=$(LIB) -shared -o $(LIB) $(LIBS) $(LIB_DIRS)
+	$(CPP) $(OBJ_C) $(OBJ_CPP) -Wl,-soname=$(LIB) -shared -o $(LIB) $(LIBS) $(LIB_DIRS)
 	$(LINK) $(LINKFLAGS) $(LIB) $(LIBPFX).so
 
 all: $(LIB)
